client/me/application-password-item/index.jsx

```diff
@@ -9,9 +9,10 @@ var React = require( 'react' ),
 /**
  * Internal dependencies
  */
-var ActionRemove = require( 'me/action-remove' ),
-	eventRecorder = require( 'me/event-recorder' ),
-	errorNotice = require( 'state/notices/actions' ).errorNotice;
+var eventRecorder = require( 'me/event-recorder' ),
+	errorNotice = require( 'state/notices/actions' ).errorNotice,
+	Gridicon = require( 'gridicons' ),
```

Comment:
`gridicons` is technically an external dep now. Do you mind moving this up to  ` External dependencies`. Same thing for the other files that import gridicons.